SDKBuildScripts/unity_RunAutoTests.sh

```diff
@@ -132,6 +137,82 @@ ExecXboxOnConsole() {
     . "$WORKSPACE/JenkinsSdkSetupScripts/JenkinsScripts/Consoles/xbox/unity_xbox.sh"
 }
 
+TryBuildAndTestAndroid() {
+    echo "TestAndroid: $TestAndroid"
+    if [ "$TestAndroid" = "true" ]; then
+        echo === Build and Test Android ===
+        pushd "${ProjRootPath}/${SdkName}_TC"
+
+            #copy test title data in
+            pushd "$WORKSPACE/SDKGenerator/SDKBuildScripts"
+                ./unity_copyTestTitleData.sh "$WORKSPACE/sdks/UnitySDK/Testing/Resources" copy
+            popd
+            if [[ $? -ne 0 ]]; then return 1; fi
+            
+            #build the APK
+            $UNITY_VERSION -projectPath "$WORKSPACE/$UNITY_VERSION/${SdkName}_TC" -quit -batchmode -executeMethod PlayFab.Internal.PlayFabPackager.MakeAndroidBuild -logFile "$WORKSPACE/${SdkName}/buildPackageOutput.txt" || (cat "$WORKSPACE/${SdkName}/buildAndroidOutput.txt" && return 1)
+            if [[ $? -ne 0 ]]; then return 1; fi
+            
+            pushd "$WORKSPACE/SDKGenerator/SDKBuildScripts"
+                
+                #pull the test title data back out
+                ./unity_copyTestTitleData.sh "$WORKSPACE/sdks/UnitySDK/Testing/Resources" delete
+                if [[ $? -ne 0 ]]; then return 1; fi
+                
+                #upload the APK and run the tests on AppCenter Test
+                ./runAppCenterTest.sh "$ProjRootPath/${SdkName}_TC/testBuilds/PlayFabAndroid.apk" "$WORKSPACE/SDKGenerator/SDKBuildScripts/AppCenterUITestLauncher/AppCenterUITestLauncher/debugassemblies" android
+                if [[ $? -ne 0 ]]; then return 1; fi
+           
+            popd
+
+            #check the cloudscript for the test results
+            JenkernaughtSaveCloudScriptResults
+
+        popd
+    fi
+}
+
+
+TryBuildAndTestiOS() {
+    echo "TestiPhone: $TestiPhone"
+    if [ "$TestiPhone" = "true" ]; then
+        echo === Build and Test iOS ===
+        
+        pushd "${ProjRootPath}/${SdkName}_TC"
+
+            #copy the test title data in
+            pushd "$WORKSPACE/SDKGenerator/SDKBuildScripts"
+                ./unity_copyTestTitleData.sh "$WORKSPACE/sdks/UnitySDK/Testing/Resources" copy
+            popd
+            if [[ $? -ne 0 ]]; then return 1; fi
+            
+            #build the xcode project to prepare for IPA generation
+            $UNITY_VERSION -projectPath "$WORKSPACE/$UNITY_VERSION/${SdkName}_TC" -quit -batchmode -executeMethod PlayFab.Internal.PlayFabPackager.MakeIPhoneBuild -logFile "$WORKSPACE/${SdkName}/buildPackageOutput.txt" || (cat "$WORKSPACE/${SdkName}/buildiPhoneOutput.txt" && return 1)
```

Comment:
I feel like this should probably point to its own folder instead of re-using _TC.

Reply:
Nope, leave it under TestClient